README.md

```diff
@@ -1 +1,193 @@
-# Rails-Capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>Rails Capstone</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+
+- [📖 Rails Capstone](#Rails-Capstone)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+
+# 📖 Rails Capstone <a name="about-project"></a>
+
+> Rails capstone is a web application. You can add categories, transactions for that categories.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>Postgresql</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- We can add, remove, update and display blogs.
+- It has user authentication.
+- It is well designed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [https://sohaib-budget.onrender.com]
+- Please use this link for icon whenever you want to create a category(https://coffee.alexflipnote.dev/random)
```

Comment:
- Hey, this is great, but this won't improve the user experience of your app. Remember the users will interact with your live demo link and not your readme file. How do users get to know about using this particular URL to always want to add an icon in your app when they can add it from their PC locally from their files.? 
- I left a comment on how to upload images in a rails app. let's implement it to improve your app and to meet the project requirement.

Reply:
### PRESENTATION VIDEO


- Hey, In your [video](https://www.loom.com/share/65c234b48c3648129eb87d4ba06b7ce3), you've not shown a highlight about a piece of the code you find interesting or are proud of making. let's add this missing feature in the video by recording another video.